<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyClientClass</class>
 <widget class="QMainWindow" name="MyClientClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>744</width>
    <height>791</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MyClient</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
QWidget#MyClientClass{

 background:qlineargradient(spread:reflect, x1:0.847, y1:0.869, x2:0.129, y2:0.12, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255))
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPlainTextEdit" name="outEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>550</y>
      <width>701</width>
      <height>221</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(172, 172, 172)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>460</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(29, 29, 29)</string>
    </property>
    <property name="text">
     <string>状态显示</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>510</y>
      <width>701</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QProgressBar{
border:2px solid grey;
border-radius:5px;
}

QProgressBar::chunk {
background-color:#05B8CC;
width:20px;
}</string>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>711</width>
      <height>321</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>文件上传</string>
     </attribute>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>40</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:rgb(58, 58, 58)</string>
      </property>
      <property name="text">
       <string>选择你要上传的文件</string>
      </property>
     </widget>
     <widget class="QToolButton" name="btnPut">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>200</y>
        <width>71</width>
        <height>61</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">color:rgb(250, 250, 250);
background:rgb(140, 140, 140);
border:2px groove gray;border-radius:10px;padding:2px 4px;</string>
      </property>
      <property name="text">
       <string>上传</string>
      </property>
      <property name="icon">
       <iconset resource="../myclient.qrc">
        <normaloff>:/MyClient/Resources/上传.png</normaloff>:/MyClient/Resources/上传.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextUnderIcon</enum>
      </property>
      <property name="autoRaise">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QToolButton" name="btnSelect">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>170</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(140, 140, 140);
border:2px groove gray;border-radius:10px;padding:2px 4px;
color:rgb(250, 250, 250);</string>
      </property>
      <property name="text">
       <string>选择文件</string>
      </property>
      <property name="icon">
       <iconset resource="../myclient.qrc">
        <normaloff>:/MyClient/Resources/选择文件 (1).png</normaloff>:/MyClient/Resources/选择文件 (1).png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextBesideIcon</enum>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineSelect">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>100</y>
        <width>231</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(172, 172, 172)</string>
      </property>
     </widget>
     <widget class="QTreeView" name="treeFile">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>41</y>
        <width>401</width>
        <height>251</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
QMainWindow#MyClient{

 background:rgb(254, 254, 254)
}</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>10</y>
        <width>201</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>显示当前目录文件列表</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>文件下载</string>
     </attribute>
     <widget class="QTreeWidget" name="treeList">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>70</y>
        <width>381</width>
        <height>201</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(172, 172, 172)</string>
      </property>
      <property name="indentation">
       <number>20</number>
      </property>
      <property name="itemsExpandable">
       <bool>true</bool>
      </property>
      <attribute name="headerDefaultSectionSize">
       <number>100</number>
      </attribute>
      <attribute name="headerMinimumSectionSize">
       <number>26</number>
      </attribute>
      <column>
       <property name="text">
        <string>文件名</string>
       </property>
       <property name="textAlignment">
        <set>AlignLeft|AlignVCenter</set>
       </property>
      </column>
      <column>
       <property name="text">
        <string>文件大小</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>文件类型</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="btnList">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>20</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:rgb(250, 250, 250);
background:rgb(172, 172, 172);
border:2px groove gray;border-radius:10px;padding:2px 4px;</string>
      </property>
      <property name="text">
       <string>显示</string>
      </property>
      <property name="icon">
       <iconset resource="../myclient.qrc">
        <normaloff>:/MyClient/Resources/显示.png</normaloff>:/MyClient/Resources/显示.png</iconset>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>50</y>
        <width>231</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:rgb(58, 58, 58)</string>
      </property>
      <property name="text">
       <string>输入你要下载的目录</string>
      </property>
     </widget>
     <widget class="QToolButton" name="btnGet">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>200</y>
        <width>61</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>7</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:rgb(250, 250, 250);
background:rgb(140, 140, 140);
border:2px groove gray;border-radius:10px;padding:2px 4px;</string>
      </property>
      <property name="text">
       <string>下载</string>
      </property>
      <property name="icon">
       <iconset resource="../myclient.qrc">
        <normaloff>:/MyClient/Resources/下载 (1).png</normaloff>:/MyClient/Resources/下载 (1).png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextUnderIcon</enum>
      </property>
     </widget>
     <widget class="QToolButton" name="btnPath">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>170</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(140, 140, 140);
border:2px groove gray;border-radius:10px;padding:2px 4px;
color:rgb(250, 250, 250);</string>
      </property>
      <property name="text">
       <string>目录切换</string>
      </property>
      <property name="icon">
       <iconset resource="../myclient.qrc">
        <normaloff>:/MyClient/Resources/文件.png</normaloff>:/MyClient/Resources/文件.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextBesideIcon</enum>
      </property>
     </widget>
     <widget class="QLineEdit" name="linePath">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>100</y>
        <width>231</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(172, 172, 172)</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>183</width>
      <height>23</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label_9">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>端口</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="linePort">
       <property name="styleSheet">
        <string notr="true">background:rgb(172, 172, 172)</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="btnCancel">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>80</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(250, 250, 250);
background:rgb(140, 140, 140);
border:2px groove gray;border-radius:10px;padding:2px 4px;
</string>
    </property>
    <property name="text">
     <string>断开</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>183</width>
      <height>23</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>主机</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineHost">
       <property name="styleSheet">
        <string notr="true">background:rgb(172, 172, 172)</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="btnConn">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>50</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(250, 250, 250);
background:rgb(140, 140, 140);
border:2px groove gray;border-radius:10px;padding:2px 4px;</string>
    </property>
    <property name="text">
     <string>连接</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRegister">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>80</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(250, 250, 250);
background:rgb(140, 140, 140);
border:2px groove gray;border-radius:10px;padding:2px 4px;</string>
    </property>
    <property name="text">
     <string>注册</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_3">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>50</y>
      <width>183</width>
      <height>23</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="label_10">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>用户</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineUser">
       <property name="styleSheet">
        <string notr="true">background:rgb(172, 172, 172)</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="btnLogin">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>50</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(250, 250, 250);
background:rgb(140, 140, 140);
border:2px groove gray;border-radius:10px;padding:2px 4px;</string>
    </property>
    <property name="text">
     <string>登录</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_4">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>80</y>
      <width>183</width>
      <height>23</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QLabel" name="label_13">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>密码</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="linePassward">
       <property name="styleSheet">
        <string notr="true">background:rgb(172, 172, 172)</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>470</y>
      <width>541</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QScrollBar" name="verticalScrollBar">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>550</y>
      <width>20</width>
      <height>151</height>
     </rect>
    </property>
    <property name="singleStep">
     <number>10</number>
    </property>
    <property name="pageStep">
     <number>50</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>744</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCaidan">
    <property name="title">
     <string>菜单</string>
    </property>
    <addaction name="actionLogin"/>
    <addaction name="actionReg"/>
    <addaction name="actionConn"/>
    <addaction name="actionCancel"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionPut"/>
    <addaction name="actionGet"/>
    <addaction name="actionList"/>
    <addaction name="actionSelect"/>
    <addaction name="actionPath"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actionHelp"/>
   </widget>
   <addaction name="menuCaidan"/>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionLogin"/>
   <addaction name="actionReg"/>
   <addaction name="actionCancel"/>
   <addaction name="actionConn"/>
   <addaction name="actionHelp"/>
   <addaction name="actionPut"/>
   <addaction name="actionGet"/>
   <addaction name="actionList"/>
   <addaction name="actionSelect"/>
   <addaction name="actionPath"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionLogin">
   <property name="icon">
    <iconset resource="../myclient.qrc">
     <normaloff>:/MyClient/Resources/登录.png</normaloff>:/MyClient/Resources/登录.png</iconset>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
   <property name="toolTip">
    <string>登录</string>
   </property>
  </action>
  <action name="actionReg">
   <property name="icon">
    <iconset resource="../myclient.qrc">
     <normaloff>:/MyClient/Resources/注册.png</normaloff>:/MyClient/Resources/注册.png</iconset>
   </property>
   <property name="text">
    <string>注册</string>
   </property>
   <property name="toolTip">
    <string>注册</string>
   </property>
  </action>
  <action name="actionCancel">
   <property name="icon">
    <iconset resource="../myclient.qrc">
     <normaloff>:/MyClient/Resources/断开链接.png</normaloff>:/MyClient/Resources/断开链接.png</iconset>
   </property>
   <property name="text">
    <string>断开</string>
   </property>
   <property name="toolTip">
    <string>断开</string>
   </property>
  </action>
  <action name="actionConn">
   <property name="icon">
    <iconset resource="../myclient.qrc">
     <normaloff>:/MyClient/Resources/连接.png</normaloff>:/MyClient/Resources/连接.png</iconset>
   </property>
   <property name="text">
    <string>连接</string>
   </property>
   <property name="toolTip">
    <string>连接</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset resource="../myclient.qrc">
     <normaloff>:/MyClient/Resources/帮助.png</normaloff>:/MyClient/Resources/帮助.png</iconset>
   </property>
   <property name="text">
    <string>帮助</string>
   </property>
   <property name="toolTip">
    <string>帮助</string>
   </property>
  </action>
  <action name="actionPut">
   <property name="icon">
    <iconset resource="../myclient.qrc">
     <normaloff>:/MyClient/Resources/上传.png</normaloff>:/MyClient/Resources/上传.png</iconset>
   </property>
   <property name="text">
    <string>上传</string>
   </property>
   <property name="toolTip">
    <string>上传</string>
   </property>
  </action>
  <action name="actionGet">
   <property name="icon">
    <iconset resource="../myclient.qrc">
     <normaloff>:/MyClient/Resources/下载 (1).png</normaloff>:/MyClient/Resources/下载 (1).png</iconset>
   </property>
   <property name="text">
    <string>下载</string>
   </property>
   <property name="toolTip">
    <string>下载</string>
   </property>
  </action>
  <action name="actionList">
   <property name="icon">
    <iconset resource="../myclient.qrc">
     <normaloff>:/MyClient/Resources/显示.png</normaloff>:/MyClient/Resources/显示.png</iconset>
   </property>
   <property name="text">
    <string>显示</string>
   </property>
   <property name="toolTip">
    <string>显示</string>
   </property>
  </action>
  <action name="actionSelect">
   <property name="icon">
    <iconset resource="../myclient.qrc">
     <normaloff>:/MyClient/Resources/选择文件 (1).png</normaloff>:/MyClient/Resources/选择文件 (1).png</iconset>
   </property>
   <property name="text">
    <string>选择</string>
   </property>
   <property name="toolTip">
    <string>选择</string>
   </property>
  </action>
  <action name="actionPath">
   <property name="icon">
    <iconset resource="../myclient.qrc">
     <normaloff>:/MyClient/Resources/文件.png</normaloff>:/MyClient/Resources/文件.png</iconset>
   </property>
   <property name="text">
    <string>文件路径</string>
   </property>
   <property name="toolTip">
    <string>文件路径</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../myclient.qrc"/>
 </resources>
 <connections/>
</ui>
